client/web/reporter/src/components/Report/Blocks/index.vue

```diff
@@ -188,10 +188,14 @@ export default {
       this.dataframes = {}
       const frames = []
 
-      this.block.elements.forEach((element) => {
+      this.block.elements.forEach((element, key) => {
         element = reporter.DisplayElementMaker(element)
 
         if (element && element.kind !== 'Text') {
+          if (element.elementID === '0') {
```

Comment:
If it works its probably by accident, since how does the element.elementID match any names if that's not whats in the block element names

Reply:
`element.reportDefinitions` will assign the element.elementID to element's name https://github.com/cortezaproject/corteza/blob/79b414ecdbe85c9080b6ae12a5b4d220aeaba23a/client/web/reporter/src/components/Report/Blocks/index.vue#L199